src/PLLModelLanguagesGetListDynamicMethodReturnTypeExtension.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Set return type of Model\Languages->get_list().
+ */
+
+declare(strict_types=1);
+
+namespace WPSyntex\Polylang\PHPStan;
+
+use WP_Syntex\Polylang\Model\Languages;
+use PHPStan\Reflection\MethodReflection;
+
+class PLLModelLanguagesGetListDynamicMethodReturnTypeExtension extends PLLModelGetLanguagesListDynamicMethodReturnTypeExtension {
```

Comment:
Should the hierarchy be the other way around?
I mean, `PLL_Model::get_language_lists()` is "kinda" deprecated so `Model\Languages::get_list()` should be the parent I guess.

Reply:
I was thinking the same actually, but didn't get through.
Done in [8711e64](https://github.com/polylang/polylang-phpstan/pull/14/commits/8711e64681dc381ae4fc81c78c2b662c8d6a2385).